pkg/asset/manifests/dns.go

```diff
@@ -91,31 +89,5 @@ func (d *DNS) Files() []*asset.File {
 
 // Load loads the already-rendered files back from disk.
 func (d *DNS) Load(f asset.FileFetcher) (bool, error) {
```

Comment:
Why gut this method instead of removing it completely?  Can't we get away with just `Files()` for use in `(*Manifests).Generate(...)` and turn `*DNS` from a `WritableAsset` into an `Asset`?  Maybe we want to split interfaces:

```go
// WritableAsset is an Asset that has files that can be written to disk.
// It can also be loaded from disk.
type WritableAsset interface {
  Asset

  // Files returns the files to write.
  Files() []*File
}

type ReadWritableAsset interface {
  WriteableAsset

  // Load returns the on-disk asset if it exists.
  // The asset object should be changed only when it's loaded successfully.
  Load(FileFetcher) (found bool, err error)
}

Reply:
I had those separate interfaces when I wrote the asset store originally. I'll have to dig back into past PRs to refresh my memory about who had what objections to that approach.

Reply:
I would like to pass on making any change along these lines for this PR. I agree with you that it is ugly to have asset with `Load` functions that do nothing. However, the problem is more widespread than just these manifest assets. I think that changes that we make to remove unused `Load` functions are worthy of their own PR.